src/track/beatsimporter.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <QVector>
+#include <memory>
+
+#include "audio/streaminfo.h"
+
+namespace mixxx {
+
+/// Importer class for Beats that can correct timing offsets when the
+/// signal info (channel number, sample rate, bitrate) is known.
+class BeatsImporter {
+  public:
+    BeatsImporter() = default;
+    virtual ~BeatsImporter() = default;
+
+    virtual bool isEmpty() const {
```

Comment:
Why not pure virtual to prevent forgetting to override this method in derived classes?